edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test.csproj

```
@@ -1,10 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
-
-    <IsPackable>false</IsPackable>
-  </PropertyGroup>
+  <Import Project="..\..\..\netcoreappVersion.props" />
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
```

please also update to use 16.4.0
```suggestion
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
```

---

And please update xunit.runner.visualstudio to 2.4.1.  Also please update other projects to use 1 single version of moq (4.13.1) in this PR or another PR.

---

I brought it back to a single 4.10 version. I wouldn't want to make solution wide changes in this PR.